src/app/shared/services/iam.service.ts

```diff
@@ -87,6 +87,10 @@ export class IamService {
     return this.signerService.providerType;
   }
 
+  get isEthSigner() {
+    return this.signerService.isEthSigner;
```

Comment:
Another question, can this be null or undefined or always is set to true/false ?

Reply:
It should be defined as long as the session is active